nodecg-io-ws-client/extension/index.ts

```diff
@@ -45,6 +49,18 @@ class WSClientService extends ServiceBundle<WSClientServiceConfig, WSClientServi
             getRawClient() {
                 return client;
             },
+            send(message: string) {
+                client.send(message);
+            },
+            onClose(func: () => void) {
+                client.on('close', func);
+            },
+            onMessage(func: (message: WebSocket.Data) => void) {
+                client.on('message', message => func(message));
+            },
+            onError(func: (error: Error) => void) {
+                client.on('error', error => func(error))
+            }
```

Comment:
Here we can pass the handler directly and don't need to create a new function that then calls the passed function. Just makes this code a little easier to understand.
```suggestion
            onMessage(func: (message: WebSocket.Data) => void) {
                client.on('message', func);
            },
            onError(func: (error: Error) => void) {
                client.on('error', func)
            }
```